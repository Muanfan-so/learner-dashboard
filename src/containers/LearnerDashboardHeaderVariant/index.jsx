import React from 'react';

import MasqueradeBar from 'containers/MasqueradeBar';
import ConfirmEmailBanner from 'containers/LearnerDashboardHeader/ConfirmEmailBanner';

import CollapsedHeader from './CollapsedHeader';
import ExpandedHeader from './ExpandedHeader';

import './index.scss';

export const LearnerDashboardHeaderVariant = () => (
  <>
    <ConfirmEmailBanner />
    <CollapsedHeader />
    <ExpandedHeader />
    <MasqueradeBar />
  </>
);

LearnerDashboardHeaderVariant.propTypes = {};

export default LearnerDashboardHeaderVariant;
