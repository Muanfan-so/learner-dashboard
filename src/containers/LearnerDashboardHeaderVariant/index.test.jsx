import { shallow } from 'enzyme';
import LearnerDashboardHeaderVariant from '.';

jest.mock('containers/LearnerDashboardHeader/ConfirmEmailBanner', () => 'ConfirmEmailBanner');
jest.mock('containers/MasqueradeBar', () => 'MasqueradeBar');
jest.mock('./CollapsedHeader', () => 'CollapsedHeader');
jest.mock('./ExpandedHeader', () => 'ExpandedHeader');

describe('LearnerDashboardHeaderVariant', () => {
  test('render', () => {
    const wrapper = shallow(<LearnerDashboardHeaderVariant />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('ConfirmEmailBanner')).toHaveLength(1);
    expect(wrapper.find('MasqueradeBar')).toHaveLength(1);
    expect(wrapper.find('CollapsedHeader')).toHaveLength(1);
    expect(wrapper.find('ExpandedHeader')).toHaveLength(1);
  });
});
