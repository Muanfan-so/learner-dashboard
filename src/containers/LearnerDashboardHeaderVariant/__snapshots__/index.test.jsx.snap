// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LearnerDashboardHeaderVariant render 1`] = `
<Fragment>
  <ConfirmEmailBanner />
  <CollapsedHeader />
  <ExpandedHeader />
  <MasqueradeBar />
</Fragment>
`;
